app/src/components/GrantDetailsRow.vue

```diff
@@ -1,89 +1,79 @@
-        <img class="m-auto" :src="logoURI || '/src/assets/logo.svg'" />
-      </div>
-      <div class="col-span-4 md:col-span-2 grid grid-cols-12 pt-8">
-        <!-- raised, contract, round, matching -->
-        <div class="text-center xl:text-left col-span-12 mb-8 xl:col-span-5 xl:col-start-2">
-          <div class="px-4 xl:px-0 xl:pt-2">
-            <div><span class="text-grey-400 mr-4">Raised:</span>{{ totalRaised }}</div>
-            <div>
-              <span class="text-grey-400 mr-4">Address:</span>
-              <a
-                class="link"
-                :href="`https://etherscan.io/address/${payoutAddress}`"
-                target="_blank"
-                rel="noopener noreferrer"
-              >
-                {{ formatAddress(payoutAddress) }}
-              </a>
-            </div>
-            <div class="xl:pt-6">
-              <span class="text-grey-400 mr-4">In Round:</span>
+  <div class="border-b border-grey-100 grid grid-cols-1 md:grid-cols-2 gap-x-8">
+    <!--grid:left (img)-->
+    <div class="bg-pink">
+      <img class="shadow-light object-cover h-full" :src="logoURI || '/src/assets/placeholder_grant.svg'" />
+    </div>
+
+    <!--grid:right (txt)-->
+    <div class="my-6">
+      <!--subgrid-->
+      <div class="grid grid-cols-1 xl:grid-cols-2 gap-8">
+        <!--subgrid:left (raised, contract, round, matching )-->
+        <div class="">
+          <!-- raised -->
+          <div><span class="text-grey-400 mr-4">Raised:</span>{{ totalRaised }}</div>
+
+          <!--contract-->
+          <div>
+            <span class="text-grey-400 mr-4">Contract:</span>
+            <a
+              class="link"
+              :href="`https://etherscan.io/address/${payoutAddress}`"
+              target="_blank"
+              rel="noopener noreferrer"
+              >{{ formatAddress(payoutAddress) }}</a
+            >
+          </div>
+
+          <!--round-->
+          <div>
+            <span class="text-grey-400 mr-4">In Round:</span>
+            <span v-for="(round, index) in roundDetails" :key="index">
+              {{ round.name }}<span v-if="index + 1 < roundDetails.length">, </span>
+            </span>
+          </div>
+
+          <!--matching-->
+          <div>
+            <span class="text-grey-400 mr-4">Matching:</span>
+            <span v-for="(round, index) in roundDetails" :key="index">
+              {{ round.matching }} {{ round.matchingToken.symbol }}
+              <span v-if="index + 1 < roundDetails.length">, </span>
+            </span>
+          </div>
+        </div>
+
+        <!--subgrid:right ( donate button, matching example )-->
+        <div class="pr-12">
+          <!-- button -->
+          <div class="text-right xl:flex xl:justify-end">
+            <button v-if="isInCart(grant?.id)" @click="removeFromCart(grant?.id)" class="btn in-cart btn-primary">
+              <CartIcon class="icon-small" />Remove
```

Comment:
The teal/green color for the remove button feels a bit confusing since removing from cart feels like a red or neutral color. IMO "Add" should be teal and "Remove" dark grey or red, or both Add and Remove should be dark grey. But @scco you're the designer here so if you disagree that's fine with me